src/main/java/org/spongepowered/api/data/manipulator/catalog/CatalogEntityData.java

```diff
@@ -304,6 +306,12 @@
      */
     public static final Class<GameModeData> GAME_MODE_DATA = GameModeData.class;
     /**
+     * Represents that an entity has a glowing outline. Few entities, such
+     * as {@link Snowball}, do not show this glow.
+     * <!-- TODO: Find all non-effected entities -->
```

Comment:
Could you remove this TODO line? From what I can tell, the fact that some entities don't glow appears to be more of an artifact of the renderer, rather than any deliberate exclusion.